cdap-kubernetes/src/main/java/io/cdap/cdap/master/environment/k8s/KubeMasterEnvironment.java

```diff
@@ -129,6 +129,7 @@ public class KubeMasterEnvironment implements MasterEnvironment {
   private static final Logger LOG = LoggerFactory.getLogger(KubeMasterEnvironment.class);
 
   public static final String SECURITY_CONFIG_NAME = "cdap-security";
+  public static final String RESOURCE_QUOTA_NAME = "cdap-resource-quota";
```

Comment:
Do we need this change?

Reply:
made this change in order to access the name in this step https://github.com/cdapio/cdap/pull/14355/files#diff-eacc77f99061f15e516c5f593374c5bc5cf06e57e12f78d851212843801d28f1R100